api/resolvers/growth.js

```diff
@@ -131,9 +131,9 @@ export default {
           json_build_object('name', 'comments', 'value', coalesce(sum(comments),0))
         ) AS data
         FROM ${viewGroup(range, 'sub_stats')}
-        WHERE sub_name='${sub}'
+        WHERE sub_name = $3
         GROUP BY time
-        ORDER BY time ASC`, ...range)
+        ORDER BY time ASC`, ...range, sub)
```

Comment:
The prior code introduced a vector for what is called [SQL injection](https://en.wikipedia.org/wiki/SQL_injection). Basically it was running raw user input against the database. Passing it as a parameter like this makes the DB sanitize it.

This isn't your fault. We make exceptions to build our dynamic query here, which we needed to do in older versions of Prisma, but it's no longer necessary (so this needs to be refactored).